### PR TITLE
Leyon Changes

### PR DESCRIPTION
**Pull request by Arrow768**
The Leyon Pistol has been changed to be closer to the original vision of it.
It is loaded via a special 10mm speedloader, however loading it takes some time.

Each gun (when ordered via cargo) will ship with a speedloader.
The speedloaders itself can not be ordered, but 10mm ammunition boxes can which can be used to restock the speedloaders or load the gun directly.

Reduces the price for the gun at the merchants from 1500 to 500 credits
And makes additional ammo boxes purchaseable

Also changes the path name of the 10mm ammo/mags/... to be consistent with the standard pathing for ammo
Adds additional messages if you can not load a gun with a single cartridge (currently it fails silently)

### CODE REVIEW

**Comment by NonQueueingMatt** [comment]
Might want to make it use the fancy new pistol gunshot sound.

File: code/modules/projectiles/guns/projectile/pistol.dm [hunk 1]
```diff
@@ -276,16 +276,18 @@
 	. = ..()
 
 /obj/item/weapon/gun/projectile/leyon
-	name = "10mm pistol"
-	desc = "The Leyon LCC Everyman is a small pistol that holds five shots and is loaded with a stripper clip, popular for self-defense on Mars. Uses 10mm rounds."
+	name = "Leyon pistol"
```
Comment (Alberyk) [inline code/modules/projectiles/guns/projectile/pistol.dm]: All guns, at least the great majority of the ballistics follow this format, the brand or etc should stay on the desc.

Comment (Arrow768) [reply code/modules/projectiles/guns/projectile/pistol.dm]: Indeed, but we already have one that is called "10mm pistol" (/obj/item/weapon/gun/projectile/tanto)

Comment (Alberyk) [reply code/modules/projectiles/guns/projectile/pistol.dm]: I guess that the name of the tanto, that can fire in burst, could be changed to indicate the automatic feature it has.

Comment (LordFowl) [reply code/modules/projectiles/guns/projectile/pistol.dm]: Yes. We already have a 10mm pistol. Not really sure why we needed another one.

Comment (Arrow768) [reply code/modules/projectiles/guns/projectile/pistol.dm]: The reasoning for it has been explained in the suggestion topic that has been created when it was originally implemented
https://forums.aurorastation.org/viewtopic.php?f=21&t=11486

**Pull request merged**